eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -71,6 +71,19 @@ stages:
                           echo "##vso[task.setvariable variable=Package.Archive]$DETECTED_PACKAGE_NAME"
                         displayName: Detecting package archive
 
+                      - pwsh: |
```

Comment:
Would be good to get this encapsulated in a little script to make it easier to test and debug.